handler/create_user.go

```diff
@@ -0,0 +1,89 @@
+package handler
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"github.com/jmoiron/sqlx"
+	_ "github.com/lib/pq"
+	"github.com/rs/zerolog/log"
+)
+
+type createUserRequest struct {
+	UserId string `json:"user_id"`
+}
+
+func CreateUser(w http.ResponseWriter, r *http.Request) {
+	ctx := r.Context()
+	var reqBody createUserRequest
+
+	// TODO: Put error handling in helper function
```

Comment:
just depends on how informative the error message needs to be. A bad request can be due to many things like the equest body is too large, invalid values, etc. If we want to be more informative in the message then we can do swithc-cases for each error kind. Tbh it would save time to not have to do this lol.

the other thing is the error from json.Decode may not always be a bad request - it could be a json unmarshaling error which in that case would be an InternServer Error not a bad request.

I don't personally care if we implement these  - just put the comment in here in case @Lev1ty was concerned about specific error handling :)

Reply:
if you don't care then i don't care either

Reply:
I think we should have everything logged so when something break, we can at least look through the logs to debug. If you think it's not worth the time right now, we can accept this tech debt and revisit error handling later.